### PR TITLE
Add Auth0-Vue v2 beta Quickstart

### PR DESCRIPTION
**Pull request by frederikprijck**
Adding the beta quickstart that goes with the soon-to-be-released v2 version of Auth0-Vue.

The only change code-wise is the placement of the returnTo parameter when calling logout, as well as mention of `openUrl` for local logout.

### CODE REVIEW

File: articles/quickstart/spa/vuejs-beta/interactive.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+::::
+
+## Add logout to your application {{{ data-action=code data-code="logout.js" }}}
+
+Users who log in to your project will also need a way to log out. When users log out, your application will redirect them to your [Auth0 logout](https://auth0.com/docs/api/authentication?javascript#logout) endpoint, which will then redirect them to the specified `authorizationParams.returnTo` parameter.
+
+Use the `logout` function exposed on the return value of `useAuth0`, which you can access in your component's `setup` function, to log the user out of your application.
+
+:::note
+To log the user out of your application but not from Auth0, use `logout({ openUrl: false })`.
```
Comment (frederikprijck) [inline articles/quickstart/spa/vuejs-beta/interactive.md:121]: Will need to replace this if we decide to go with a different name.

File: articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  <div>
+    <button @click="logout">Log out</button>
+  </div>
+</template>
+
+<script>
+  export default {
+    methods: {
+      logout() {
+        this.$auth0.logout({ authorizationParams: { returnTo: window.location.origin } });
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md]:
```suggestion
        this.$auth0.logout({ logoutParams: { returnTo: window.location.origin } });
```

File: articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+<script>
+  import { useAuth0 } from '@auth0/auth0-vue';
+
+  export default {
+    setup() {
+      const { logout } = useAuth0();
+
+      return {
+        logout: () => {
+          logout({ authorizationParams: { returnTo: window.location.origin } });
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md]:
```suggestion
          logout({ logoutParams: { returnTo: window.location.origin } });
```

File: articles/quickstart/spa/vuejs-beta/files/logout.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  import { useAuth0 } from '@auth0/auth0-vue';
+
+  export default {
+    setup() {
+      const auth0 = useAuth0();
+
+      return {
+        logout() {
+          auth0.logout({ 
+            authorizationParams: {
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/files/logout.md]:
```suggestion
            logoutParams: { 
```

File: articles/quickstart/spa/vuejs-beta/files/logout.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      };
+    }
+  };
+
+  // Options API
+  export default {
+    methods: {
+      logout() {
+        this.$auth0.logout({ 
+          authorizationParams: {
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/files/logout.md]:
```suggestion
          logoutParams: { 
```

File: articles/quickstart/spa/vuejs-beta/interactive.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+Run your application and click the logout button, verify that:
+* your Vue application redirects you to the `authorizationParams.returnTo` address
+* you are no longer logged in to your application
+
+:::
+
+:::checkpoint-failure
+If you are not able to logout:
+* Verify you specified a value for `authorizationParams.returnTo` when calling `logout`
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/interactive.md]:
```suggestion
* Verify you specified a value for `logoutParams.returnTo` when calling `logout`
```

File: articles/quickstart/spa/vuejs-beta/interactive.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+::::checkpoint
+
+:::checkpoint-default
+
+You should now be able to log in using Auth0 Universal Login.
+
+Click the login button and verify that:
+* your Vue application redirects you to the Auth0 Universal Login page
+* you can log in or sign up
+* Auth0 redirects you to your application using the value of the `redirect_uri` you used to configure the plugin.
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/interactive.md]: Do you think we should consistently refer to `authorizationParams.redirect_uri` throughout the QS?

**Review by ewanharris — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/vuejs-beta/interactive.md (hunk 7 above)
Comment (frederikprijck) [reply articles/quickstart/spa/vuejs-beta/interactive.md]: Ye, I would

Comment (frederikprijck) [inline articles/quickstart/spa/vuejs-beta/interactive.md]:
```suggestion
* Auth0 redirects you to your application using the value of the `authorizationParams.redirect_uri` you used to configure the plugin.
```

File: articles/quickstart/spa/vuejs-beta/interactive.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+### Using the Options API
+With the Options API, you can use the same `logout` method from the global `$auth0` property through the `this` accessor.
+
+::::checkpoint
+
+:::checkpoint-default
+
+Run your application and click the logout button, verify that:
+* your Vue application redirects you to the `authorizationParams.returnTo` address
```
Comment (ewanharris) [inline articles/quickstart/spa/vuejs-beta/interactive.md]: Overlooked this one before sorry!
```suggestion
* your Vue application redirects you to the `logoutParams.returnTo` address
```

**Review by ewanharris — CHANGES_REQUESTED** [review]

**Review by ewanharris — APPROVED** [review]

**Pull request merged**